ang/civicase/ActivityCard--Long.html

```diff
@@ -1,6 +1,6 @@
 <div
   class="civicase__activity-card civicase__activity-card--long"
-  ng-click="viewInPopup($event, activity)"
+  ng-click="feedViewFn ? feedViewFn(activity.id, $event) : viewInPopup($event, activity)"
```

Comment:
Why is this necessary? The click event should be mentioned outside of the card. For example
```
  <div
              case-activity-card="activity"
              mode="long"
              refresh-on-checkbox-toggle="true"
              bulk-allowed="bulkAllowed"
              ng-click="viewActivity(activity.id, $event)"
              refresh-callback="refreshCase">
            </div>
```

Reply:
`viewActivity` is of the outside the activity scope so it can be called **on** the card. But `viewInPopup` is inside the activity card scope so can't be called inside but need to be called inside. 
Now if we call the `viewInPopup ` inside then it will break outer ng click call and vice versa. So had this implementation.